### PR TITLE
[TS] LPS-138944 Result is not fully highlighted 

### PR DESCRIPTION
**Pull request by joshuacords**
https://issues.liferay.com/browse/LPS-138944

Removed behavior of checking for updated content after returning from search. Trying to support this behavior guarantees there will be cases where we won't match the search engine's highlighting. Additionally, JournalArticles seem to be the only assets trying to do this. If an index is stale because of some type of outage, it needs to be reindexed.

I've also tested against the tickets that originated the highlighting change and did not experience any regression bugs:
- https://issues.liferay.com/browse/LPS-72222
- https://issues.liferay.com/browse/LPS-72670
- https://issues.liferay.com/browse/LPS-74642

Author: @joshuacords

### CODE REVIEW

**Comment by joshuacords** [comment]
ci:test:relevant

**Comment by joshuacords** [comment]
ci:test:relevant

**Comment by joshuacords** [comment]
com.liferay.journal.asset.test.JournalArticleAssetSearchTest.testOrderByCreateDateAsc is listed on our failures spreadsheet, I also tested locally where it passed.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/util/DDMIndexerImpl.java [hunk 1]
```diff
@@ -370,28 +372,31 @@ public String extractIndexableAttributes(
...
 				else if (value instanceof Date) {
 					sb.append(dateFormat.format(value));
-					sb.append(StringPool.SPACE);
 				}
 				else if (value instanceof Date[]) {
 					Date[] dates = (Date[])value;
 
 					for (Date date : dates) {
 						sb.append(dateFormat.format(date));
-						sb.append(StringPool.SPACE);
```
Comment (BryanEngler) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/util/DDMIndexerImpl.java:386]: i think you would still need spaces between each date right?

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/util/DDMIndexerImpl.java [hunk 2]
```diff
@@ -370,28 +372,31 @@ public String extractIndexableAttributes(
...
 					}
 				}
 				else if (value instanceof Object[]) {
 					Object[] values = (Object[])value;
 
 					for (Object object : values) {
 						sb.append(object);
-						sb.append(StringPool.SPACE);
```
Comment (BryanEngler) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/util/DDMIndexerImpl.java:394]: same here

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/util/DDMIndexerImpl.java [hunk 3]
```diff
@@ -359,6 +359,8 @@ public String extractIndexableAttributes(
 
 		Fields fields = toFields(ddmStructure, ddmFormValues);
 
+		boolean firstField = true;
```
Comment (BryanEngler) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/util/DDMIndexerImpl.java:362]: is the purpose of using this firstField variable just a source formatting thing, or are you actually trying to change the value that sb contains?

Comment (BryanEngler) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/util/DDMIndexerImpl.java:362]: i guess you are just trying to remove the trailing space

**Comment by BryanEngler** [comment]
@joshuacords i made a few tweaks and im running ci here https://github.com/BryanEngler/liferay-portal/pull/617, i didnt manually test any of it so maybe you can take a look and see if you agree with the changes etc

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/util/DDMIndexerImpl.java (hunk 2 above)
Comment (joshuacords) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/util/DDMIndexerImpl.java:394]: Yep

**Comment by BryanEngler** [comment]
see https://github.com/BryanEngler/liferay-portal/pull/617

**Pull request closed without merging**